base/MAPL_sun_uc.F90

```diff
@@ -56,6 +57,53 @@ module MAPL_SunMod
   integer, public, parameter :: MAPL_SunDailyMean       = 5
   integer, public, parameter :: MAPL_SunAnnualMean      = 6
 
+! Default solar orbital system parameters (private).
+! Dont change these unless you know what you are doing.
+! They are appropriate for the current modern epoch circa 2000.
+! -------------------------------------------------------------
+ 
+   ! Parameters of old orbital system (tabularized intercalation cycle)
+   ! ------------------------------------------------------------------
+   real, parameter    :: DEFAULT_ORBIT_ECCENTRICITY     = 0.0167
+   real, parameter    :: DEFAULT_ORBIT_OBLIQUITY        = 23.45
+   real, parameter    :: DEFAULT_ORBIT_PERIHELION       = 102.0
+   integer, parameter :: DEFAULT_ORBIT_EQUINOX          = 80
+
+   ! Parameters of new orbital system (analytic two-body), which allows some
+   ! time-varying behavior, namely, linear variation in LAMBDAP, ECC, and OBQ.
+   ! -------------------------------------------------------------------------
+
+   ! Fixed anomalistic year length in mean solar days
+   real, parameter    :: DEFAULT_ORB2B_YEARLEN          = 365.2596
+
+   ! Reference date and time for orbital parameters
+   ! (defaults to J2000 = 01Jan2000 12:00:00 TT = 11:58:56 UTC)
+   integer, parameter :: DEFAULT_ORB2B_REF_YYYYMMDD     = 20000101
+   integer, parameter :: DEFAULT_ORB2B_REF_HHMMSS       = 115856
+
+   ! Orbital eccentricity at reference date
+   real, parameter    :: DEFAULT_ORB2B_ECC_REF          = 0.016710
+   ! Rate of change of orbital eccentricity per Julian century
+   real, parameter    :: DEFAULT_ORB2B_ECC_RATE         = -4.2e-5
+
+   ! Earth's obliquity (axial tilt) at reference date [degrees]
+   real, parameter    :: DEFAULT_ORB2B_OBQ_REF          = 23.44
+   ! Rate of change of obliquity [degrees per Julian century]
+   real, parameter    :: DEFAULT_ORB2B_OBQ_RATE         = -1.3e-2
+
+   ! Longitude of perihelion at reference date [degrees]
+   !   (from March equinox to perihelion in direction of earth's motion)
+   real, parameter    :: DEFAULT_ORB2B_LAMBDAP_REF      = 282.947
+   ! Rate of change of LAMBDAP [degrees per Julian century]
+   !   (Combines both equatorial and ecliptic precession)
+   real, parameter    :: DEFAULT_ORB2B_LAMBDAP_RATE     = 1.7195
+
+   ! March Equinox date and time
+   ! (defaults to March 20, 2000 at 07:35:00 UTC)
+   integer, parameter :: DEFAULT_ORB2B_EQUINOX_YYYYMMDD = 20000320 
+   integer, parameter :: DEFAULT_ORB2B_EQUINOX_HHMMSS   =  73500
+
```

Comment:
You might want to document the units for these where it makes sense.

Reply:
@tclune see the units in the square brackets for my ORB2B parameters.